gm4_speed_paths/overlay_26/data/gm4_speed_paths/functions/main.mcfunction

```diff
@@ -0,0 +1,6 @@
+# Clock is started by tick advancement, on_path, only active while necessary
+# run from function, gm4_speed_paths:clock/start, or scheduled by self
+
+execute as @a at @s run function gm4_speed_paths:apply_effects/path_check
```

Comment:
If the goal is to optimize this module, then I don't think running a function `as @a` regardless of score or tag is not that smart. Is there no check you can add to this?

Reply:
Previously it was 2 @a's, one that checks the tag and one that doesn't.  That effectively covers all of @a, and this reduces it to 1